packages/domainslib/domainslib.0.4.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "KC Sivaramakrishnan <kc@kcsrk.info>"
+authors: ["KC Sivaramakrishnan <kc@kcsrk.info>"]
+homepage: "https://github.com/ocaml-multicore/domainslib"
+doc: "https://ocaml-multicore.github.io/domainslib/"
+synopsis: "Parallel Structures over Domains for Multicore OCaml"
+license: "ISC"
+dev-repo: "git+https://github.com/ocaml-multicore/domainslib.git"
+bug-reports: "https://github.com/ocaml-multicore/domainslib/issues"
+tags: ["org:ocamllabs"]
+depends: [
+  "dune" {>= "1.8"}
```

Comment:
```suggestion
  "ocaml" {>= "5"}
  "dune" {>= "1.8"}
```
We need to make it explicit that it is incompatible with 4.12+domains

Reply:
Don't know if "5" is the right number there though

Reply:
Thanks! I've followed the `5.00` convention used here https://github.com/ocaml/opam-repository/blob/master/packages/base-domains/base-domains.base/opam#L7. Might probably need an update later if the version numbering were to change.